crates/ruff_linter/src/rules/pyupgrade/rules/pep695/non_pep695_type_alias.rs

```diff
@@ -40,6 +40,8 @@ use super::{
 ///
 /// ## Example
 /// ```python
+/// from typing import TypeAlias, TypeAliasType
```

Comment:
Let's add the other missing imports too, even if they're not necessary to trigger UP040 out of the box:

```suggestion
/// from typing import TypeAlias, TypeAliasType, Annotated
/// from annotated_types import Gt
```
 
And could you also add the missing imports of `Annotated` and `Gt` to the `Use instead:` block below?